src/UI/citizens/components/submit-request/CitizensSubmitRequest.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import {
+  Container,
+  Title,
+  Krug,
+  Broj,
+  Mail,
+  Adresa,
+  Mobitel,
+  Kontakt,
+  Opis,
+  Nesto,
+  Button2
+} from './CitizensSubmitRequestStyle';
+
+const CitizensSubmitRequest = () => (
+  <Container>
+    <link
+      rel="stylesheet"
+      media="screen"
+      href="https://fontlibrary.org/face/poppins"
+      type="text/css"
+    />
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="//fonts.googleapis.com/css?family=Lora"
+    />
+
+    <Krug />
+    <Broj>4</Broj>
+
+    <Title>Kraj</Title>
+
+    <Opis>
+      Nakon što ste ispunili sve podatke, pošaljite svoj zahtjev na razmatranje.
+      Povratnu informaciju ćete dobiti u <p>što kraćem roku.</p>
```

Comment:
Zašto je samo "što kraćem roku" wrappano <p> elementom? Taj dio teksta nije ništa posebniji od ostatka. Sve mora biti unutar paragrafa